### DISCUSSION TITLE
阿里云盘被限制登陆，不知道与webdav是否有关

### DISCUSSION THREAD

**Discussion by jalog** [messense/aliyundrive-webdav#727] [Q&A]
前几天阿里云盘突然掉线，再次登陆提示“限制登陆”
经过电子邮件向客服申诉，得到以下回复：
您好，系统检测到您的账号系统访问行为异常，可能在违规使用系统功能。为确保您的账号安全，已开启保护模式冻结账号。
如再次被检测到系统访问行为异常等违规使用系统功能，您的账号将会被永久冻结，请规范使用阿里云盘。

系统访问行为异常，不知道具体指的是什么，提醒朋友们注意。

**Comment by jalog** [comment]
![image](https://user-images.githubusercontent.com/24936408/221352218-f9c51f77-fd09-4fca-9629-ef3829c55ab8.jpeg)

**Comment by AceXa11** [comment]
是不是用了小白羊多线程下载的问题?

**Comment by messense** [comment]
> 系统访问行为异常

一般来说跟具体软件无关，主要是看使用方式是否涉嫌滥用，比如短时间大量下载、上传文件占用大量带宽资源影响云盘服务稳定性或者其他用户使用体验，利用云盘做黑灰产等。

**Comment by oyksoft** [comment]
我昨天也被封了，估计是用了alist挂载的原因。阿里云盘是最不靠谱的，动不动就封锁用户账号，用户数据一点都无法保证。
